Automattic/vip-go-mu-plugins
# Search Dev Tools: address some edge cases

## Description

During the staging testing we've discovered a couple of issues:

1. Dev Tools were loaded regardless of whether the Search itself is properly initialized or not that could cause a fatal error. I.e. if the Search code is enabled but the Search itself is not configured properly (constants are not defined correctly) we don't initialize Search.
2. Very long strings could break the layout.
3. As a bonus, updating some dependencies.

## Changelog Description

### Plugin Updated: Search Dev Tools

A hotfix that addressed potential layout issues and an edge case that may cause a fatal error.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out the PR.
2.  Comment out VIP_ELASTICSEARCH_ENDPOINTS or VIP_ELASTICSEARCH_USERNAME 
3. Verify there are no fatals and dev tools code isn't being loaded.
